e2e-tests/src/npg/psp-sorting.npg.integration.test.ts

```diff
@@ -137,9 +137,10 @@ describe.only("Checkout show and sort psp list", () => {
     })
     const pspEditButton = await page.waitForSelector(pspEditButtonSelector);
     await pspEditButton.click();
-    await new Promise(r => setTimeout(r, 500));
-    const pspOriginalElements = await page.$$(".pspFeeName");
-
+    
+    //Initially is sorted by fee ascending
+    await page.waitForSelector(".pspFeeValue");
```

Comment:
@scaminati-bv why did you remove the usage of pspFeeName,
double check might be a mistake

Reply:
Because initially the list is sorted by fee value. In the second check after click i check name order.

Reply:
ok, right. good catch

Reply:
now that i think about it thoug, i dont think the fee sorting should be there in the sort by name test,
i updated code here to only test sort by name eeaa717922bdc361291fda294171eda29c063593